help/en/releasenotes/current-draft-note.shtml

```diff
@@ -398,6 +398,12 @@
           <li></li>
         </ul>
 
+    <h3>LogixNG</h3>
+        <a id="LogixNG" name="LogixNG"></a>
+        <ul>
+          <li>The action Listen On Beans is improved.</li>
+        </ul>
```

Comment:
@bobjacobsen 
When I added this, I noticed that there where no LogixNG section in the current-draft-note.shtml file. Maybe LogixNG isn't added to the template for the current-draft-note.shtml.

Reply:
> When I added this, I noticed that there where no LogixNG section in the current-draft-note.shtml file. Maybe LogixNG isn't added to the template for the current-draft-note.shtml.

It's OK to add it.  The release process will copy the section over to the master file at the next test release.

Reply:
Yes, but my point is that there will probably be many new updates to LogixNG onwards. So it would be good to have the LogixNG section added to the template for the current-draft-note.shtml. But I don't know which the template file is.

Reply:
You can expand that section in current-draft-note.shtml until the next test release (today, but not the point). At the test release, current-draft-note.shtml gets reset.  But part of that reset is a diff to check for new sections being added. When there is one, like this one, it gets carried forward into the next one, and so on.  So this new section will be there, but empty, after tonights test release.